QuizAppNew/View/QAview.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  QAview.swift
+//  QuizAppNew
+//
+//  Created by Jannatun Nahar Papia on 20/8/22.
+//
+
+import SwiftUI
+
+//MARK: Question Answer view
+struct QAview: View {
+    //outgoing
```

Comment:
Why parameters such as correct, wrong, answered, set, data why need it. You are only responsible for making UI design.

Reply:
removed